### ISSUE TITLE
autosummary does not create the description for descriptor attributes

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1444]
For example, I want to document the following:

```
#!python

# MyModule.py

class D(object):
    """Descriptor class
    """

    def __get__(self, instance, owner):
        return 1

    def __set__(self, instance, value):
        pass

class A(object):
    """My Class
    """

    #: Descriptor attribute
    abc = D()
```

And I create the following RST file:

```
#!

.. module:: MyModule

.. autoclass:: A
    :members:

    .. autosummary::

        abc
```

`autosummary` creates table with the following description for attribute `abc`:

`int(x=0) -> int or long`

But in this case I want to see:

`Descriptor attribute`

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1444
- Originally reported by: Evgeny Prilepin
- Originally created at: 2014-04-09T09:29:16.620

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-04-09 23:55:41+00:00_

I confirmed the issue by attached project.

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-04-09 23:59:11+00:00_

`issue1444.patch` is a quick hacked patch. Please check the behavior.

**Comment by shimizukawa** [comment]
_From Evgeny Prilepin on 2014-04-10 05:55:16+00:00_

I applied `issue1444.patch` to current sphinx v1.3a0. Your test project is ok. But on my work project is not working. I get the following error:

```
Making output directory...
Running Sphinx v1.3a0
loading pickled environment... not yet created
building [html]: targets for 5 source files that are out of date
updating environment: 5 added, 0 changed, 0 removed
reading sources... [ 20%] app_signals

Exception occurred:
  File "C:\Programs\Python27_x86\lib\site-packages\docutils\statemachine.py", line 1158, in __setitem__
    raise TypeError('assigning non-ViewList to ViewList slice')
TypeError: assigning non-ViewList to ViewList slice
```

See attached full traceback `sphinx-err-s8a91q.log`.

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-04-10 06:14:31+00:00_

Thanks! I'll take a look soon.

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-04-10 09:33:42+00:00_

2nd quick hack (1 line changed)

Could you please check a new patch?
I'm going to test with a little more patterns later.

**Comment by shimizukawa** [comment]
_From Evgeny Prilepin on 2014-04-10 11:42:18+00:00_

Thanks! The latest patch `issue1444-2.patch` works correctly on my project.

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-04-10 11:54:08+00:00_

Thanks for your confirmation!

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-04-11 15:27:28+00:00_
- Fix: autosummary does not create the description from attributes docstring. Closes #1444

→ <<cset 8f69e25c85020e2bf9f6e1bf38d268489ab17270>>
